docs/paper/dev/getting-started/userdev.mdx

```diff
@@ -24,6 +24,12 @@ This can make it hard to work with in a development environment. This plugin let
 types, names, and fields during development, and then remaps your plugin, so it can still be used with the obfuscated
 server.
 
+:::caution[1.20.5]
+
+As of Minecraft version 1.20.5 Paper will be shipped with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings as we have traditionally done since Paper 1.17.
```

Comment:
```suggestion
As of Minecraft version 1.20.5, Paper will be shipped with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings as we have traditionally done since Paper 1.17.
```